README.md

```diff
@@ -1,10 +1,50 @@
-# dd-trace-go
-
 [![CircleCI](https://circleci.com/gh/DataDog/dd-trace-go/tree/master.svg?style=svg)](https://circleci.com/gh/DataDog/dd-trace-go/tree/master)
 [![Godoc](http://img.shields.io/badge/godoc-reference-blue.svg?style=flat)](https://godoc.org/github.com/DataDog/dd-trace-go/tracer)
 
-A Go tracing package. Currently requires at least Go 1.7.
+A Go tracing package for Datadog APM.
+
+## Contributing Quick Start
+
+Requirements:
+
+* Go 1.7 or later
+* Docker
+* Rake
+* gometalinter – `go get -u github.com/alecthomas/gometalinter`
+* golint – `go get -u github.com/golang/lint/golint`
+* goconst – `go get -u github.com/jgautheron/goconst/cmd/goconst`
+* gas – `go get -u github.com/GoASTScanner/gas`
+* ineffassign – `go get -u github.com/gordonklaus/ineffassign`
+* gocyclo – `go get -u github.com/fzipp/gocyclo`
+
+### Run the tests
+
+Start the containers defined in `docker-compose.yml`:
+
+```
+$ docker-compose up -d
+```
+
+Fetch this package's dependencies:
+
+```
+$ rake get
```

Comment:
Based on the information from @kmshultz below, pinning dependencies makes sense IMHO. Croissant just moved from glide to dep, but let's wait on feedback from @palazzem.